extension/package.json

```
@@ -18,23 +18,23 @@
     "commands": [
       {
         "command": "auditor.transform",
-        "title": "Update reviews to latest commit"
+        "title": "acg-auditor: Update reviews to latest commit"
```

Same thing for the other lines to match the repo name.
```suggestion
        "title": "auditor: Update reviews to latest commit"
```